Scripts/Generator/RoomGenerator.cs

```diff
@@ -50,10 +91,10 @@ public override void _Ready()
     /// <param name="origin">The center of the base of the room.</param>
     /// <param name="roomSize">The width and height of the room, inclusive of walls.</param>
     /// <param name="front">The front direction the room is facing</param>
-    public void GenerateRoom(Vector3 origin, Vector2 roomSize, Vector3 front)
+    public void GenerateRoom()
     {
-        Debug.Assert(roomSize.Length() > 0, "Room size cannot be zero.");
-        Debug.Assert(wallThickness * 2 <= Math.Min(roomSize.x, roomSize.y), "Walls cannot be thicker than half the width/height of the room (there must be space within the room).");
+        //Debug.Assert(roomSize.Length() > 0, "Room size cannot be zero.");
```

Comment:
Still need?

Reply:
This thing like causing the program to crash sia